x-pack/plugins/cases/server/client/utils.ts

```diff
@@ -506,6 +507,8 @@ export const sortToSnake = (sortField: string | undefined): SortFieldCase => {
     case 'closedAt':
     case 'closed_at':
       return SortFieldCase.closedAt;
+    case 'title':
```

Comment:
Do you mind if you rename the function form `sortToSnake` to `getSortField` or similar? The function does not convert from camel case to snake case.